mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/kb-credentials.feature

```diff
@@ -3,155 +3,566 @@ Feature: KB Credentials
   Background:
     * url baseUrl
     * callonce login testUser
-    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+    * configure headers = { 'Content-Type': 'application/vnd.api+json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain, application/vnd.api+json' }
+    * def credentials = read('classpath:domain/mod-kb-ebsco-java/features/samples/credentials/credentials.json')
+    * def random_uuid = 'dee3b52e-f37e-44f6-9d9c-681799c97543'
+    * def long_name = 'This name longer than 255 characters___This name longer than 255 characters___This name longer than 255 characters___This name longer than 255 characters___This name longer than 255 charactersThis name longer than 255 characters___This name longer than 255 characters___'
 
- ################ Positive test cases ################
-  @Undefined
+  @Positive
   Scenario: GET /eholdings/kb-credentials should return all existed KB credentials (response: status 200 and json body)
-    * print 'undefined'
+    Given path '/eholdings/kb-credentials'
+    When method GET
+    Then status 200
+    And match responseType == 'json'
+    And match response.data[0].type == 'kbCredentials'
 
-  @Undefined
-  Scenario: GET /eholdings/user-kb-credential should return KB credentials by given assigned user (response: status 200 and json body)
-    * print 'undefined'
-
-  @Undefined
+  @Positive
   Scenario: POST /eholdings/kb-credentials should create KB credentials (response: status 201 and json body)
```

Comment:
You should check that the credentials entity was really created with data that you sent..